### ISSUE TITLE
Build fails in exp 52.0.0

### ISSUE THREAD

**Issue by roblingle** [expo/expo#1651]
### Environment

```
# exp diagnostics
09:53:05 [exp] Generating diagnostics report...
09:53:05 [exp] You can join our slack here: https://slack.expo.io/.

Environment:
  OS: macOS High Sierra 10.13.4
  Node: 8.9.1
  Yarn: 1.5.1
  npm: 5.7.1
  Watchman: 4.7.0
  Xcode: Xcode 9.3 Build version 9E145
  Android Studio: 3.0 AI-171.4408382

Packages: (wanted => installed)
  expo: ^21.0.0 => 21.0.3
  react: 16.0.0-alpha.12 => 16.0.0-alpha.12
  react-native: https://github.com/expo/react-native/archive/sdk-21.0.2.tar.gz => 0.48.4

Diagnostics report:
https://exp-xde-diagnostics.s3.amazonaws.com/heelapp-94344192-a7e5-4fca-8e1c-f8ed52547386.tar.gz
```

### Steps to Reproduce

1. With exp 52.0.0 and in the root directory of the project: `exp build:android` or `exp build:ios`

### Expected Behavior

Build should succeed, as it still does in 49.2.2.

### Actual Behavior

`/Users/ville/Projects/expo/universe/dev/exp/src/exp.js` is not a path on my system and the error does not occur in my project itself.

```
# exp build:android
17:18:28 [exp] Making sure project is set up correctly...
17:18:33 [exp] Your project looks good!
17:18:33 [exp] Checking if current build exists...

17:18:34 [exp] No currently active or previous builds for this project.
17:18:35 [exp] Unable to find an existing exp instance for this directory, starting a new one...
17:18:43 [exp] Starting Metro Bundler on port 19001.

/Users/ville/Projects/expo/universe/dev/exp/src/exp.js:135
      for (let line of msg.split('\n')) {
                           ^
TypeError: msg.split is not a function
    at logLines (/Users/ville/Projects/expo/universe/dev/exp/src/exp.js:135:28)
    at logWithLevel (/Users/ville/Projects/expo/universe/dev/exp/src/exp.js:202:11)
    at /Users/ville/Projects/expo/universe/dev/exp/src/exp.js:256:11
    at Array.forEach (<anonymous>)
    at PackagerLogsStream.updateLogs [as _updateLogs] (/Users/ville/Projects/expo/universe/dev/exp/src/exp.js:255:22)
    at func (/xdl/src/logs/PackagerLogsStream.js:428:12)
    at PackagerLogsStream._enqueueFlushLogsToAdd (/xdl/src/logs/PackagerLogsStream.js:440:7)
    at PackagerLogsStream._enqueueAppendLogChunk (/xdl/src/logs/PackagerLogsStream.js:422:12)
    at PackagerLogsStream._handleMetroEvent (/xdl/src/logs/PackagerLogsStream.js:257:10)
    at Object.write (/xdl/src/logs/PackagerLogsStream.js:194:18)
```

**Comment by fson** [comment]
Thank you for reporting this! The bug should be fixed in `exp@52.0.3`, which I just released.

**Comment by roblingle** [comment]
Confirmed. Thanks!
